sbin/pfperl-api-docker-wrapper

```diff
@@ -10,6 +11,8 @@ args="$args -v /usr/local/pf/html/captive-portal/profile-templates:/usr/local/pf
 args="$args --privileged -v /run/systemd/system:/run/systemd/system -v /var/run/dbus/system_bus_socket:/var/run/dbus/system_bus_socket"
 args="$args -v /usr/local/fingerbank/conf:/usr/local/fingerbank/conf"
 args="$args --network=host"
+args="-e HOST_OS"
```

Comment:
args="-e HOST_OS" -> args="$args -e HOST_OS"

Reply:
Good catch.
I fixed typo and I also removed this instruction inside api-frontend-docker-wrapper because `pf/util.pm`  is not available in the image.
Can be merged if everything is correct for you @fdurand